apps/prairielearn/src/lib/server-jobs.js

```diff
@@ -335,33 +342,33 @@ module.exports.spawnJob = function (options, callback) {
       cwd: job.options.working_directory,
       env: job.options.env,
     };
-    job.proc = child_process.spawn(job.options.command, job.options.arguments, spawnOptions);
+    const proc = child_process.spawn(job.options.command, job.options.arguments, spawnOptions);
```

Comment:
Moving from the job object to a local made TypeScript happy. Nothing but this function accessed `job.proc`, so this is safe.